test.js

```diff
@@ -1,51 +1,86 @@
 const net = require('net');
 const test = require('ava');
-const pify = require('pify');
```

Comment:
It looks like we don't need this lib in deps anymore.

Could you please remove it completely?

Reply:
Done.